kitsune/wiki/tests/test_views.py

```diff
@@ -684,3 +685,144 @@ def test_it_works(self):
+        en_content = 'This artcile is in English'
+        trans_content = 'This artcile is translated into es'
+        assert en_content not in doc_content
+        assert trans_content in doc_content
+
+    def test_custom_locale_mapping(self):
+        """Test while the document is not translated into requested locale but
+        We have custom wiki fallback for the requested locale.
+        The Custom wiki fallback locale mentioned in kitsune/wiki.config.py"""
+
+        # Create a translated document in bn-BD locale and try to get bn-IN version
+        # of the document. While we the client have no language choices.
+        doc_content = self.get_data_from_translated_document(header=None,
+                                                             create_doc_locale='bn-BD',
+                                                             req_doc_locale='bn-IN')
+        # As the Document is not translated into bn-IN and there is no locale of
+        # ACCEPT_LANUGAE header which the document to, But there is a custom wiki fallback
+        # mapping available for bn-IN. the fallback locale is bn-BD. So the user should
+        # see the document in bn-BD.
+        en_content = 'This artcile is in English'
+        trans_content = 'This artcile is translated into bn-BD'
+        assert en_content not in doc_content
+        assert trans_content in doc_content
+
+    def test_custom_locale_mapping_for_header_locale(self):
+        """Test while the document is not translated into any of the locale of ACCEPT_LANGUAGE
+        Header. But we have fallback locale mentioned in Custom wiki fallback locale for one
+        of the locale of ACCEPT_LANGUAGE Header"""
+
+        # While the requested locale also in custom wiki fallback locale but none of the
+        # fallback locale for requested locale is translated
+
+        # A Client with ca, pt-PT, ja language choices in ACCEPT_LANGUAGE Header
+        header = 'ca,pt-PT;q=0.7,ja;q=0.3,'
+        # Create a translated document in pt-BR and try to get ca locale version of the
+        # document by using above mentioned client.
+        doc_content = self.get_data_from_translated_document(header=header,
+                                                             create_doc_locale='pt-BR',
+                                                             req_doc_locale='ca')
+        # As the ca version of the document is not translated moreover the we have
+        # custom locale fallback mapping for ca locale, but the document is translated to none
+        # of the fallback locale of ca. But in the header, there is pt-PT locale and we have
+        # custom wiki locale mapping for pt-PT. the fallback locale is pt-BR. and the document is
+        # translated into pt-BR, so the user should see pt-BR vesion of the document.
+        en_content = 'This artcile is in English'
+        trans_content = 'This artcile is translated into pt-BR'
```

Comment:
You have a mispelling here (and in many other places). "artcile" should be "article". Please fix this everywhere it occurs instead of repeating the spelling mistake to make the tests pass.